database.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+const User = require('./models/User');
+
+let connect = (dbUrl) => {
```

Comment:
const instead of let would be better

Reply:
Noted, I will make it const in the next iteration.